### PR TITLE
Refactor/check connection febe

### PR DESCRIPTION
**Pull request by GitHubEason**
Move health check to a new page
Screen Shot 2022-06-18 at 12 09 16 am

### CODE REVIEW

**Review by jennawan — APPROVED** [review]
Works fine on my side, nice job!

File: src/pages/health.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from "react";
+import { Flex, Button } from "@chakra-ui/react";
+import { createAsyncThunk, createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { api } from "../utils/axios";
+import { useStoreDispatch } from "../store/hooks";
+
+const healthCheck = createAsyncThunk("Health Check", async () => {
+  api(process.env.NEXT_PUBLIC_HEALTH_API as string, { method: "get" })
+    .then((response) => {
+      console.log(response);
```
Comment (jennawan) [inline src/pages/health.tsx:10]: Considering removing the console.log?

**Review by jennawan — APPROVED** [review]

File: .env.example [hunk 2]
```diff
@@ -1,4 +1,3 @@
-NEXT_PUBLIC_UAT_URL=
-NEXT_PUBLIC_TEST_API=
+NEXT_PUBLIC_DESIGN_URL=
```
Comment (tatoMa) [inline .env.example:1]: Why it is called DESIGN URL? I thought it's for the home page(marketing page) URL.

Comment (GitHubEason) [reply .env.example:1]: I think  this URL for the 404-page redirects to the design page.

Comment (GitHubEason) [reply .env.example:1]: Thanks for reminding me

**Review by tatoMa — COMMENTED** [review]
No test file for this page yet so it cannot pass the coverage request.

**Comment by Jzeva** [comment]
If you are sure that you don't need to add tests or exclude it in the test config then it looks OK to me.

**Comment by kritomomo** [comment]
I don't think the front end needs this health check

**Pull request closed without merging**